src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java

```diff
@@ -131,9 +132,9 @@ public static void importTexturesBlockOrItem(MCreator fr, BlockItemTextureSelect
 			String namec = RegistryNameFixer.fix(FilenameUtilsPatched.removeExtension(hoh.getName()));
 			File file;
 			if (type == BlockItemTextureSelector.TextureType.BLOCK) {

```

Comment:
You could also remove BlockItemTextureSelector.TextureType at this point and instead use TextureType in these places too